### DISCUSSION TITLE
Help - Account Name No Longer Visible (Simple Fix?)

### DISCUSSION THREAD

**Discussion by AllOnMyOhm** [YetiForceCompany/YetiForceCRM#14575] [How to] [answered]
I'm hoping somebody might be able to help me figure out why my "Account Name" field has disappeared.

It shows in the picklist but not when trying to add a new Account.

Am I missing a dependecy, or mapping point? I've taken screenshots of the current settings
If somebody could show me what theirs looks like perhaps I can figure out what is going wrong.

Thanks in advance
![1 - Not Visible When Adding](https://user-images.githubusercontent.com/67898192/115579450-17b29e80-a2be-11eb-966b-acb560732255.PNG)
![2 - Shows In Picklist](https://user-images.githubusercontent.com/67898192/115579455-18e3cb80-a2be-11eb-9e18-ead6221c044a.PNG)
![3- No Dependecy](https://user-images.githubusercontent.com/67898192/115579464-1aad8f00-a2be-11eb-80da-3e5080d29ded.PNG)
![4 - No Mapping Information](https://user-images.githubusercontent.com/67898192/115579472-1bdebc00-a2be-11eb-8f3b-ab86954f92c2.PNG)

**Comment by rskrzypczak** [comment, marked as answer]
Maybe you have hidden this field in this panel: https://gitdeveloper.yetiforce.com/index.php?parent=Settings&module=FieldsDependency&view=List
![image](https://user-images.githubusercontent.com/10221999/115661182-59365e80-a33d-11eb-8f90-f5b75d9c2040.png)

**Reply by AllOnMyOhm** [reply]
That's the ticket! for some reason it was inactive, now it's active it's working again. 
Thank you!
